services/galley/src/Galley/API/Public.hs

```diff
@@ -152,6 +152,10 @@ instance
 instance ToSchema a => ToSchema (Headers ls a) where
   declareNamedSchema _ = declareNamedSchema (Proxy @a)
 
+-- FUTUREWORK: Make a PR to the servant-swagger package with this instance
+instance ToSchema Servant.NoContent where
+  declareNamedSchema _ = declareNamedSchema (Proxy @())
+
```

Comment:
This is not normally necessary, because servant-swagger has a more specific instance for `Verb ... NoContent`. Here we need it because `NoContent` is inside `UVerb`, but I'm not sure this is the best way.

I think this is fine for now, but for the purposes of submitting a PR, I think the proper way to support `NoContent` in `UVerb` is to have some more specific instances for `ToSwagger` that do not necessarily require `ToSchema`. I have done something similar for `HasServer` and `HasClient` in my [servant PR](https://github.com/haskell-servant/servant/pull/1420).